### ISSUE TITLE
Add DELIMITER $$ in MySQL/MariaDB stored procedure/function DDL

### ISSUE THREAD

**Issue by yoni-yad2** [dbeaver/dbeaver#18465]
### Description

In MySQL/MariaDB DB Navigator
SP DDL Output is missing syntax "DELIMITER $$" at beginning and 
"$$ 
DELIMITER;"
at the end, in order for the SP creation to succeed.

### DBeaver Version

22.3.0.202212041619

### Operating System

Windows 10 64 bit

### Database and driver

MySQL 5.7

### Steps to reproduce

Right click on any MySQL stored procedure/function in DB navigator->Generate SQL->DDL

**Comment by E1izabeth** [comment]
Please, provide an example of such procedure DDL.

**Comment by yoni-yad2** [comment]
DELIMITER $$
CREATE PROCEDURE sp_test()
BEGIN
select 'test';
END
$$

**Comment by LonwoLonwo** [comment]
I do not think that this is a bug. A delimiter is optional. In DBeaver, you can execute this query with highlighting.
We can add Delimiter as an option to Generate SQL.
But in this case, is not clear - what delimiter do we need to use? DELIMITER // - as in [documentation](https://mariadb.com/kb/en/create-procedure/)? Or DELIMITER $$?

**Comment by yoni-yad2** [comment]
Please add it as the default option so the DDL can run successfully in an SQL editor without highlighting.
You may choose whatever DELIMITER you want (I agree to stick to // as in the official doc: https://dev.mysql.com/doc/refman/8.0/en/create-procedure.html)

*You already use $$ in the create new procedure screen

**Comment by JTMosaic** [comment]
If anything, the behavior should at least be consistent.

Expand a MySQL database  and select a procedure. Press F4 to view the procedure and make an inconsequential change.

Now, click the "Open in Console..." button at the bottom right corner of the screen. The code opens in a new editor window **_without_** the delimiter or DROP PROCEDURE statement.

However, if you instead click the "Save..." button and then click "Open Editor" the code opens in a new editor window **_with_** both the delimiter set and the DROP PROCEDURE statement.
